src/tests/unit_tls.cpp

```diff
@@ -968,7 +1030,7 @@ Test::Result test_tls_alert_strings()
       seen.insert(str);
       }
 
-   Botan::TLS::Alert unknown_alert = Botan::TLS::Alert(Botan::secure_vector<uint8_t>{01, 66});
+   Botan::TLS::Alert unknown_alert = Botan::TLS::Alert(Botan::secure_vector<uint8_t> {01, 66});
```

Comment:
Seems `vector<T>{r}` gets a space `vector<T> {r}`

This change is not Wrong... but I'm not sure it's Right either.